torchao/dtypes/utils.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict, Callable, Any
+from collections import defaultdict
+import functools
+
+# torch_function and torch_dispatch operator dispatch registrations
+_ATEN_OP_OR_TORCH_FN_TABLE: Dict[Callable, Dict[Any, Any]] = defaultdict(dict)
```

Comment:
Nit: is this typing right?

Reply:
yeah, but I can change `Any` to `Callable` as well to be more specific I think